app/src/main/res/values/strings.xml

```diff
@@ -25,6 +26,8 @@
     <string name="game_menu_state_slot_empty">Empty</string>
     <string name="game_menu_state_slot_saved">Saved: %1$s</string>
     <string name="game_menu_close_game">Close Game</string>
+    <string name="sustained_performance_mode">Sustained Performance Mode</string>
+    <string name="sustained_performance_mode_summary">May improve performance</string>
```

Comment:
"May make performance more consistent" remember that turning it on made it slower for you, people will turn it on thinking they'll get a boost and then they'll get a bust